File: js/programmers/level.0/prgm_120868_1.js
function solution(sides) {
  let result = 0;
  const min = Math.min(...sides);
  const max = Math.max(...sides);

  for (x = max - min + 1; x <= max; x++) {
    result += 1;
  }
  for (x = max + 1; x < min + max; x++) {
    result += 1;
  }
  return result;
}

File: python/programmers/level.0/prgm_120868_1.py
def solution(sides):
    result = 0
    min_num = min(sides)
    max_num = max(sides)

    for i in range(max_num - min_num + 1, max_num + 1):
        result += 1
    
    for j in range(max_num + 1, min_num + max_num):
        result += 1
        

    return result